lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * dummyLayoutV1.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien. 
+ */
+
+// Create a graph instance.
+
+
+function buildGraph() {
+  var lib = './../';
```

Comment:
I didn't bring this up during the meeting, but I wanted to mention that we should start trying to add "use strict" as the first line of our explicit function declarations. This forces the javascript run time to be stricter about what syntax is allowed.

This is actually a good thing since it means that we can catch errors before they become bugs.